consulagent/consulKVStore.go

```diff
@@ -93,8 +93,13 @@ func (ca *ConsulAgent) GetKeyValuePairs(prefix string) map[string][]byte {
 // GetValue gets the value of the key
 func (ca *ConsulAgent) GetValue(key string) []byte {
 	pair, _, err := ca.consulAgent.KV().Get(key, nil)
+	if pair == nil {
```

Comment:
should first check for error...then for nil. whenever there is an error the pair will be nil